Tests/XCUITests/TopTabsTest.swift

```diff
@@ -468,17 +468,15 @@ class TopTabsTestIpad: IpadOnlyTestCase {
         app.buttons["TopTabsViewController.newTabButton"].tap()
         app.buttons["TopTabsViewController.newTabButton"].tap()
         waitForExistence(app.buttons["TopTabsViewController.tabsButton"])
-        let numTab = app.buttons["Show Tabs"].value as? String
-        XCTAssertEqual("3", numTab)
+        waitForExistence(app.buttons["Show Tabs"].staticTexts["3"], timeout: TIMEOUT)
+        waitForValueContains(app.buttons["Show Tabs"], value: "3")
```

Comment:
Unfortunately, `waitForValueContains` does not accept a `timeout` argument.

Would the command `waitForExistence` sufficient and the `waitForValueContains` is not needed?

Reply:
Looking at the test failure: 
  **XCTAssertEqual failed: ("Optional("2")") is not equal to ("Optional("3")")**  
I don't think the error is here since we get the 3 tabs open.

The problem is below in line:                                                                  
full-functional-ipad/Tests/XCUITests/TopTabsTest.swift:479

So, I would say, we are not removing correclty one tab by clicking on the 'x' button or we are not waiting enough to check the tab counter when that happens...

Also, I don't think this command works: `waitForValueContains(app.buttons["Show Tabs"], value: "3")`
I had to do a `value as? String` otherwise you see (Optional("3") instead of just "3"

Reply:
Yes, I know that the failure occurs a few lines later, but I'd like to make the checks on `app.buttons["Show Tabs"]` the same throughout the same test.

Rather than converting a value to a `String`, I would like to use a command that is from the language to compare the values. `waitForValueContains` works on my end but I can't find the documentation for `waitForValueContains` from https://developer.apple.com.

In any case, I would like to `wait` until the value desired appears in `app.buttons["Show Tabs"]` because there's an animation updating the number.

Reply:
The test is never failing for me on checking the number of tabs. It is failing on removing the tabs via the 'x' button. 
I can't remove the tabs by clicking on tha button on the simulator... there might be a bug there ...

Reply:
We've confirmed that clicking the "x" from the tab does not close the tab on iOS 16.0 only. Clicking the same "x" under iOS 16.1 works.

Reply:
@isabelrios With the known issue, could we upgrade the test environment and monitor the test?